run_tests.py

```diff
@@ -1,42 +1,16 @@
 import os
 import sys
-from pathlib import Path
-from tests.fixtures import (
-    TEST_SERIES_FIELDS,
-    TEST_DWI_SERIES_FIELDS,
-    TEST_PATIENT_FIELDS,
-)
-
 import django
+import shutil
 from django.conf import settings
 from django.test.utils import get_runner
 
 
-def clean_media():
-    media = (
-        Path(settings.MEDIA_ROOT)
-        / "MRI/DICOM"
-        / TEST_PATIENT_FIELDS["uid"]
-        / TEST_SERIES_FIELDS["uid"]
-    )
-    media_dwi = (
-        Path(settings.MEDIA_ROOT)
-        / "MRI/DICOM"
-        / TEST_PATIENT_FIELDS["uid"]
-        / TEST_DWI_SERIES_FIELDS["uid"]
-    )
-    media.rmdir()
-
-    while str(media_dwi) != settings.MEDIA_ROOT:
-        media_dwi.rmdir()
-        media_dwi = media_dwi.parent
```

Comment:
What do you think about:
```python
from django.core.exceptions import ImproperlyConfigured

def clean_media():
    media = Path(settings.MEDIA_ROOT)
    if 'tests' in media:
        shutil.rmtree(media)
    else:
        raise ImproperlyConfigured("...")
```
?

Reply:
Actually, if I understand correctly, this function isn't at all used anymore, is it?
It's possible that it would be better to replace the `shutil.rmtree(settings.IMPORTED_PATH)` below with something similar to the propsed `clean_media()` above, just to try and prevent an edge case in which someone somehow runs this function in the real environment and deletes the entire media directory.

Reply:
I can't think of a situation where this sort of thing will be possible, but I have no problem with this change.